src/main/java/com/minecolonies/coremod/client/gui/WindowHutAllInventory.java

```diff
@@ -135,7 +135,7 @@ private void updateResources()
     {
         final List<BlockPos> containerList = building.getContainerList();
 
-        final List<ItemStack> items = new ArrayList<>();
+        final Map<ItemStorage, ItemStorage> storedItems = new HashMap<>();
         final World world = building.getColony().getWorld();
```

Comment:
Yes better, and make this a Map<ItemStorage, Integer> and increment the quantity in the. And keep ItemStorage the same

Reply:
Did that.